t/apicast-policy-standalone.t

```diff
@@ -32,3 +32,37 @@ GET /t HTTP/1.1
 --- no_error_log
 [error]
 [warn]
+
+
+
+=== TEST 2: standalone can point to external destination
+--- environment_file: standalone
+--- configuration_format: yaml
+--- configuration
+server:
+  listen:
+  - port: $TEST_NGINX_SERVER_PORT
+    name: test
```

Comment:
Everything is named `test`. It's not obvious to know what attributes should have matching names.

Reply:
Fair point. Will make better names.